web/ui/src/common/jellyTable.js

```diff
@@ -131,68 +131,59 @@
                             sortedItems,
                             tableEntries;
 
-                        if (allItems === undefined) {
-
-                            // if just intitalized, show loading and default to empty array
-                            allItems = [];
-                            toggleNoData(false);
-                            return;
-
-                        } else if (angular.isObject(allItems) && !angular.isArray(allItems)) {
-
-                            // single-entry tables that are not arrays pass through once (eg config files)
+                        if (angular.isObject(allItems) && !angular.isArray(allItems)) {
+                            // single-entry tables that are not arrays and not null
```

Comment:
this comment is not correct. This block is hit if the data is an object, and is not necessarily single-entry.